changelog.md

```diff
@@ -19,7 +19,9 @@
 * None.
 
 ### New Features
-* None.
+* A new `AuthProviderConfig` object can now hold the auth config exposed by EWB
+* A new helper functions to create AuthProviderConfig by fetching the data from EWB, and to fetch provider-related configuration (jwkUri, tokenEndpoint)
+* `ZepbenTokenFetcher` is now using the new `AuthProviderConfig` object to fetch the auth config and the token.
```

Comment:
just realized this is in the wrong section? ... this doco is being added to 0.9.0 not 0.10.0. Also, aren't there technically some breaking changes? Ie public method signatures have changed.